src/scroll-timeline-base.js

```diff
@@ -577,24 +590,33 @@ export class ViewTimeline extends ScrollTimeline {
 
   // Currently specced as fit with proposal to rename in order to more naturally
   // support start and end transitions.
+  // http://github.com/w3c/csswg-drafts/issues/7044
+  // TODO: Update once specced.
   get range() {
     return scrollTimelineOptions.get(this).range;
   }
 
   get currentTime() {
     const unresolved = null;
-    if (this.phase === 'inacitve')
+    if (this.phase === 'inactive')
       return unresolved;
 
     // Compute the offset of the top-left corner of subject relative to
     // top-left corner of the container.
     const container = this.source;
     const target = this.subject;
 
-    const targetBounds = target.getBoundingClientRect();
-    const containerBounds = container.getBoundingClientRect();
-    const top = targetBounds.top - containerBounds.top;
-    const left = targetBounds.left - containerBounds.left;
+    // const targetBounds = target.getBoundingClientRect();
+    // const containerBounds = container.getBoundingClientRect();
+
+    let top = 0;
+    let left = 0;
+    let node = target;
+    while (node != container) {
```

Comment:
Note, we should avoid an infinite loop here in case container is not actually an offsetParent of node. We should probably also throw an error when you attempt to use such a container.

Reply:
There should never be an infinite loop since a node cannot be its own offsetParent, but you could hit a null offsetParent.

Regarding throwing an exception, should this be specced?  It would seem a bit spammy to throw on each scroll position update.  Perhaps it should be caught during style update.  Alternatively, the phase should simply be inactive.  I'm implemented that later for now with a TODO to revisit. Also filed a spec issue.